charts/monitoring/templates/alertmanager.yaml

```diff
@@ -0,0 +1,13 @@
+---
+apiVersion: monitoring.coreos.com/v1beta1
+kind: AlertmanagerConfig
+metadata:
+  name: {{ .Values.name }}-alerting
+spec:
+  route:
+    receiver: default
+  receivers:
+  - name: default
+    webhookConfigs:
+      - url: >-
+          https://el-ct-listener-{{ .Release.Namespace }}-mlops.{{ .Values.cluster_domain }}/
```

Comment:
are this the alertmanager webhook also for the uwm?

Reply:
this is specifically for [uwm](https://github.com/rhoai-mlops/deploy-lab/blob/main/charts/templates/trustyai/enable_uwm.yaml#L9):) but we use the same alertmanager, I don't want people to bother creating a specific alertmanager..the existing one can take the load of the exercises :)